config/locales/en.yml

```diff
@@ -157,7 +157,7 @@ en:
         teacher_training_adviser/steps/subject_like_to_teach:
           attributes:
             preferred_teaching_subject_id:
-              invalid_type: "Please select maths, physics or modern foreign language"
+              inclusion: "Please select maths, physics, modern foreign language or other"
```

Comment:
Perhaps "Please select a subject" would be easier to grasp here? Or "Please tell us which subject you'd like to teach". Thoughts @croftla?

Reply:
Choose a subject or other